src/components/message/MessageItem.jsx

```
@@ -2,26 +2,39 @@ import React from 'react';
 import { formatTime } from '../../utils/formatTime';
 import ReservationCompleteMessage from '../message/ReservationCompleteMessage';
+import ReservationDecisionMessage from '../message/ReservationDecisionMessage';
 import DecisionCard from '../message/DecisionCard';
+import { useAuth } from '../../context/AuthContext';
 
 export default function MessageItem({ msg, isMine }) {
   if (!msg?.isReservationCard && !msg?.message?.trim()) {
     return null;
   }
+
+  const { auth } = useAuth();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)